imap_processing/cdf/global_attrs.py

```diff
@@ -8,12 +8,15 @@
 This check occurs during the xarray_to_cdf step, which has an example in
```

Comment:
@bryan-harter This isn't something we'd want to change in this PR, but are there any attributes we should add in here to conform to SPASE?

Reply:
Eventually we'll probably want to add an attribute for the spase_DatasetResourceID, but it's not actually necessary.  We can discuss the SPASE requirement more at the SDWG tomorrow, but the internal format of our CDF files doesn't need to be anything special to comply to SPASE